stoplight.js

```diff
@@ -0,0 +1,23 @@
+//Readme instructions for Jquery.
+
+var red = $("#red.bulb")
+var yellow = $("#yellow.bulb")
+var green = $("#green.bulb")
+
+function changeLightRed(){
+	$("#stopLight").css("background", "red");
+}
+
+function changeLightYellow(){
+	$("#stopLight").css("background", "yellow");
+}
+
+function changeLightGreen(){
+	$("#stopLight").css("background", "green");
+}
+
```

Comment:
you can add the `removeClass()` method to all these functions and target each of the light container divs before you change the css using the existing classes.